packages/core/src/components/IconButton/IconButton.module.scss

```diff
@@ -6,4 +6,16 @@
 
 .referenceWrapper {
   display: inline-flex;
-}
\ No newline at end of file
+}
+
+.loader.loader {
+  width: 16px;
+  height: 16px;
+  &.xxs {
+    width: 10px;
+    height: 10px;
+    svg {
+      display: block;
+    }
```

Comment:
Why is it necessary? Is it inline by default?

Reply:
![image](https://github.com/mondaycom/monday-ui-react-core/assets/104433616/ff05bbd7-44a4-4f49-b4f1-5e9178618495)
It's inline by default, yeah
Without display block it isn't displaying properly